### PR TITLE
TW-1434 Save auth to collection updates

### PR DESCRIPTION
**Pull request by claireriley**
beta preview: https://tw-1434-save-auth-to-collection.learning.postman-beta.com/docs/sending-requests/authorization/authentication-for-public-apis/

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/sending-requests/authorization/authentication-for-public-apis.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ contextual_links:
...
 
-<img alt="Postman API set up new authorization" src="https://assets.postman.com/postman-docs/v10/postman-api-easier-auth-v10-2.jpg"/>
+<img alt="Postman API set up new authorization" src="https://assets.postman.com/postman-docs/v10/set-up-new-auth-v10.20.jpg"/>
 
 Select **Set up new authorization** to learn what authentication credentials you'll need to make your request and where you can find them. For APIs that support OAuth 2.0, you can select **Authorize** to complete authorization. For APIs requiring credentials like tokens or API keys, you can paste those directly into your request.
 
-<img alt="Postman API authentication info and link" src="https://assets.postman.com/postman-docs/v10/postman-api-easier-auth-enter-key-v10-2.jpg"/>
+<img alt="Postman API authentication info and link" src="https://assets.postman.com/postman-docs/v10/save-auth-to-collection-v10.20.jpg"/>
+
+If you're creating an individual request, you can select **Save authorization** to save your credentials to an environment for reuse. If you're working in a collection, you can select **Save authorization to collection**, which will let you use it for other requests in your collection.
```
Comment (bhargavkaranam) [inline src/pages/docs/sending-requests/authorization/authentication-for-public-apis.md:44]: @claireriley We allow saving auth to collection only if request is inheriting auth from parent. Can we tweak this to include that?

**Review by gitThingsDone — APPROVED** [review]
+1, on @bhargavkaranam's comment, otherwise LGTM

**Pull request merged**